bot/cogs/commands/useful.py

```diff
@@ -636,9 +636,29 @@ async def image_search(self, ctx: Ctx, *, query):
         await ctx.reply_embed(ctx.l.useful.search.nope)
         return
 
-    @commands.command(name="remindme", aliases=["remind"])
+    @commands.group(name="reminder", aliases=["remind"])
     @commands.cooldown(1, 2, commands.BucketType.user)
-    async def remind_me(self, ctx: Ctx, duration_str: str, *, reminder: str):
+    async def reminder_parent(self, ctx: Ctx):
```

Comment:
We could make the method name `reminders` but idrc

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/265/commits/1c1b2257f93b36758f63d3aa4374c14ef75e0a96